src/lib.rs

```diff
@@ -6,6 +6,7 @@
 // See `LICENSE` in this repository.
 
 use std::io::{self, Read};
+use std::{panic, process};
 
 /// Utility that reads a `Vec` of bytes from standard input (stdin)
```

Comment:
todo: update doc comments to mention that using these functions treats panics as aborts